From 73a3e1a1f57ffebeb3a0e4b1c6459a800a2e6ff9 Mon Sep 17 00:00:00 2001
From: Gleb Mazovetskiy <glex.spb@gmail.com>
Date: Sat, 23 Jan 2021 00:14:36 +0000
Subject: [PATCH] Patch install dir

---
 src/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index b290b96..9691221 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -215,9 +215,9 @@ else()
     endif()
     install(TARGETS GL
       LIBRARY
-      DESTINATION "/usr/lib/gl4es/"
+      DESTINATION "/usr/lib/openmw/gl4es/"
     )
     install(FILES "../include/gl4esinit.h" "../include/gl4eshint.h"
-      DESTINATION "/usr/include/gl4es/"
+      DESTINATION "/usr/include/openmw/gl4es/"
     )
 endif()
-- 
2.27.0

