From 8e079d9ee33883032fb2ddd319bbf6130e70028d Mon Sep 17 00:00:00 2001
From: Gleb Mazovetskiy <glex.spb@gmail.com>
Date: Tue, 12 Jan 2021 21:20:53 +0000
Subject: [PATCH] Patch install dir

---
 src/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 835d059..d63e5f2 100755
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -211,10 +211,10 @@ else()
     endif()
     install(TARGETS GL
     LIBRARY
-      DESTINATION "/usr/lib/gl4es/"
+      DESTINATION "/usr/lib/openmw/gl4es/"
     )
   install(FILES "../include/gl4esinit.h" "../include/gl4eshint.h"
-    DESTINATION "/usr/include/gl4es/"
+    DESTINATION "/usr/include/openmw/gl4es/"
     )
   endif()
 
-- 
2.27.0

