From 1bd340b8bf41c7c56daee12d57a48f9122995e16 Mon Sep 17 00:00:00 2001
From: Gleb Mazovetskiy <glex.spb@gmail.com>
Date: Wed, 13 Jan 2021 19:53:47 +0000
Subject: [PATCH 4/4] cmake: OSG_STATIC: Link OSG dependencies in the static
 build

---
 apps/openmw/CMakeLists.txt | 14 ++++----------
 1 file changed, 4 insertions(+), 10 deletions(-)

diff --git a/apps/openmw/CMakeLists.txt b/apps/openmw/CMakeLists.txt
index 65061f6d5..714605827 100644
--- a/apps/openmw/CMakeLists.txt
+++ b/apps/openmw/CMakeLists.txt
@@ -150,6 +150,9 @@ target_link_libraries(openmw
 )
 
 if(OSG_STATIC)
+    # OSG dependencies must be linked in the static build:
+    target_link_libraries(openmw dl z freetype fontconfig jpeg png)
+
     # We use `--whole-archive` because plugins use registration.
     set(_osg_plugins -Wl,--whole-archive ${_osg_libraries_used_by_osg_plugins})
     foreach(PLUGIN_NAME ${USED_OSG_PLUGINS})
@@ -162,16 +165,7 @@ else()
 endif(OSG_STATIC)
 
 if (ANDROID)
-    target_link_libraries(openmw
-        EGL
-        android
-        log
-        dl
-        z
-        freetype
-        jpeg
-    	png
-    )
+    target_link_libraries(openmw EGL android log)
 endif (ANDROID)
 
 if (USE_SYSTEM_TINYXML)
-- 
2.27.0

